stacks/portstack.go

```diff
@@ -167,7 +167,7 @@ func (ps *PortStack) RecvEth(ethernetFrame []byte) (err error) {
 	if len(payload) < eth.SizeEthernetHeader+eth.SizeIPv4Header {
 		return errPacketSmol
 	} else if len(payload) > int(ps.mtu) {
-		println("recv", payload, ps.mtu)
+		//println("recv", payload, ps.mtu)
```

Comment:
Better to remove altogether if modifying

Reply:
global searched for println's all out